src/main/java/bio/terra/workspace/app/controller/GlobalExceptionHandler.java

```diff
@@ -60,10 +61,10 @@
   }
 
   private ResponseEntity<ErrorReport> buildErrorReport(
-      Throwable ex, HttpStatus statusCode, List<String> causes) {
+      @NotNull Throwable ex, HttpStatus statusCode, List<String> causes) {
```

Comment:
I added this b/c `ex.getMessage()` is flagged as possibly throwing an NPE. I can be talked out of it.